schema/denm/denm_schema_2-1-0.json

```diff
@@ -0,0 +1,579 @@
+              "altitude": {
+                "type": "integer",
+                "description": "Unit: 0.01 meter. referenceEllipsoidSurface(0), oneCentimeter(1), unavailable(800001)",
+                "default": 800001,
+                "minimum": -100000,
+                "maximum": 800001
+              }
+            }
+          },
+          "message_type": {
+            "description": "root message type source of the element",
+            "enum": [
+              "denm",
+              "cam",
+              "cpm",
+              "po"
+            ]
+          }
+        }
+      }
+    },
+    "message": {
+      "type": "object",
+      "required": [
+        "protocol_version",
+        "station_id",
+        "management_container"
+      ],
+      "properties": {
+        "protocol_version": {
+          "type": "integer",
+          "description": "version of the ITS message and/or communication protocol",
+          "minimum": 0,
+          "maximum": 255,
+          "examples": [
+            1
+          ]
+        },
+        "station_id": {
+          "type": "integer",
+          "description": "identifier for an ITS-S",
+          "minimum": 0,
+          "maximum": 4294967295,
+          "examples": [
+            1,
+            42
+          ]
+        },
+        "management_container": {
+          "type": "object",
+          "required": [
+            "action_id",
+            "detection_time",
+            "reference_time",
+            "event_position"
+          ],
+          "properties": {
+            "action_id": {
+              "type": "object",
+              "required": [
+                "originating_station_id",
+                "sequence_number"
+              ],
+              "properties": {
+                "originating_station_id": {
+                  "type": "integer",
+                  "description": "identifier of an its station",
+                  "minimum": 0,
+                  "maximum": 4294967295
+                },
+                "sequence_number": {
+                  "type": "integer",
+                  "description": "The sequence number is set each time a new DENM is created. It is used to differentiate from events detected by the same ITS-S.",
+                  "minimum": 0,
+                  "maximum": 65535
+                }
+              }
+            },
+            "detection_time": {
+              "type": "integer",
+              "description": " Time at which the event is detected by the originating ITS-S. For the DENM repetition, this DE shall remain unchanged.",
+              "minimum": 0,
+              "maximum": 4398046511103,
+              "$comment": "Remains the same on repetition; updated if any value changed in the payload, meaning sequence_number remains unchanged but anything else has been updated",
+              "$comment": "Unit: millisecond since ETSI epoch (2004/01/01, so 1072915200000); utcStartOf2004(0), oneMillisecAfterUTCStartOf2004(1)"
```

Comment:
Double comment: include the unit part in the description (actually, I think that even the other comment could be included in the secription).

Reply:
I actually separated the unit from the description to ease readability
Otherwise we already have 50km long string which are a pain to read
I can still put it unit back to the description if it is preferred

But it might fit a bit more the semantic of those fields

Reply:
In the other messages we put the units in the description field (with sometimes very long descriptions, which can be relevant in my opinion to have a self-explaining schema). In any case, some JSON schema parsers flag the double comments as a warning.